lib/pbench/agent/tool_meister.py

```diff
@@ -80,6 +80,16 @@
 fmtstr = "%(asctime)s %(levelname)s %(process)s %(thread)s %(name)s %(funcName)s %(lineno)d -- %(message)s"
 
 
+def log_subprocess_output(
+    pipe: subprocess.PIPE, logger: logging.Logger, name: str
+) -> logging.Logger:
+    """Thread start function to log outputs from a given pipe.
+    """
```

Comment:
Sure, but why add support for the general case when we don't need it?

Reply:
Because the only change is to the type hint?

(I.e., thanks to the magic which is Python, we already **have** the support...we just have to "let" callers use it.)